Documentation/PageTsconfig/TcaDefaults.rst

```diff
@@ -33,3 +33,7 @@ Example:
 
    # Show newly created pages by default
    TCAdefaults.pages.hidden = 0
+
+..  note::
+    `TCAdefaults` is not (yet) applied to FlexForm values. These can only be addressed via
+    actual :xml:`<default>` elements within the FlexForm DataStructure.
```

Comment:
I would omit "actual", or is there something special you like to emphasize?

Reply:
And one more: DataStructure as in XMLs "<DataStructure>" or as in "data structure"?